Fix missing include statement.

Patch author: bernd.schubert@fastmail.fm

diff --git a/src/string.h b/src/string.h
--- a/src/string.h
+++ b/src/string.h
@@ -6,6 +6,8 @@
 
 #ifndef UNIONFS_STRING_H
 #define UNIONFS_STRING_H
+
+#include <string.h>
 
 char *whiteout_tag(const char *fname);
 int build_path(char *dest, int max_len, ...);
